src/hooks/useLotto.ts

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import { LOTTO_PRICE } from "../constants/lotto";
+import { generateLottoNumbers } from "../lib/lotto";
+import type { LottoNumbersSet } from "../types/lotto";
+
+const useLotto = () => {
+  const [lottoNumbersSet, setLottoNumbersSet] = useState<LottoNumbersSet>([]);
```

Comment:
p4: 개인적으로는 이렇게 표현하는 게 더 직관적일 것 같아요.
```suggestion
  const [lottoNumbersSet, setLottoNumbersSet] = useState<LottoNumbers[]>([]);
```

Reply:
좋은 의견인 것 같습니다 👍🏾👍🏾